apps/web-giddh/src/app/reports/components/report-dashboard/reports.dashboard.component.html

```diff
@@ -1,3 +1,4 @@
+<hamburger-menu [pageHeading]="'Report'"></hamburger-menu>
```

Comment:
This should be Reports Since it's Report on All Modules Page

Reply:
Done